### PR TITLE
Replace plasma cutter EMP effect

### LINKED ISSUE #43685: Plasma Cutters are affected by EMP
They are recharged by plasma and not electricity. Seems silly that the loose all their charge from EMP

### PR DESCRIPTION
**Pull request by ThatReformedEngineer**
🆑 
add: plasma cutters have a new emp effect
/🆑

fixes #43685

### CODE REVIEW

File: code/modules/projectiles/guns/energy/special.dm [hunk 1]
```diff
@@ -159,6 +159,16 @@
 	else
 		..()
 
+/obj/item/gun/energy/plasmacutter/emp_act(severity)
+	if(!cell.charge)
+		return
+	cell.use(cell.charge/3)
+	if(isliving(loc))
+		var/mob/living/L = loc
+		L.visible_message("<span class='danger'>Searing plasma is spewed out of [L]'s [src]!</span>", "<span class='userdanger'>Your [src] malfunctions and spews incinerating plasma onto you!</span>")
```
Comment (4dplanner) [inline code/modules/projectiles/guns/energy/special.dm]: "Incinerating plasma" is a bad phrase imo

**Comment by ThatReformedEngineer** [comment]
Yeah i am not creative I with words

File: code/modules/projectiles/guns/energy/special.dm (hunk 1 above)
Comment (ThatReformedEngineer) [reply code/modules/projectiles/guns/energy/special.dm]: Yeah I am god dam horrid with words to be honest

**Comment by ATH1909** [comment]
There should probably be a message that's visible to everyone in visual range about plasma leaking out of the plasma cutter, even if there's nobody holding the plasma cutter.

File: code/modules/projectiles/guns/energy/special.dm (hunk 1 above)
Comment (chaoko99) [reply code/modules/projectiles/guns/energy/special.dm]: Also "searing plasma spews from" is better phrasing here.

**Comment by Anonmare** [comment]
Make an armour save of 4+ for the Gets Hot rule

File: code/modules/projectiles/guns/energy/special.dm (hunk 1 above)
Comment (ExcessiveUseOfCobblestone) [reply code/modules/projectiles/guns/energy/special.dm]:
```suggestion
		L.visible_message("<span class='danger'>Concentrated plasma discharges from [src] onto [L], burning them!</span>", "<span class='userdanger'>Your [src] malfunctions, spewing concentrated plasma onto you! It burns!</span>")
```

**Comment by optimumtact** [comment]
Hello?

**Comment by ExcessiveUseOfCobblestone** [comment]
> losing PR to a string

**Pull request closed without merging**